public/css/style.css

```diff
@@ -190,3 +190,38 @@ p {
   font-size: 1.5vw;
   color: black;
 }
+
+/* div with the button to scroll down to the bottom of the page */
+#scroll_down {
+  display: flex;
+  justify-content: center;
+}
+
+#scroll_down_button {
```

Comment:
Hi @Aravindh-SNR  If you see the  `#scroll_down_button`  and `#scroll_up_button` ,both these `ids` have the same styles properties so you can just create a single `class` say : `scroll-btn` with these same properties and then use that class in instead of those two `ids` .Its always good practice to create any reusable code.

Reply:
@RocktimSaikia Ah, yes. I totally missed that. Thanks for pointing it out!